src/test/java/com/epam/dataservice/DaytimeTypeServiceTest.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.dataservice;
+
+import static org.junit.Assert.assertEquals;
+
+import java.time.LocalTime;
+
+import com.epam.data.RoadAccident.DAYTIMETYPE;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DaytimeTypeServiceTest {
+	private static DaytimeTypeService testService;
+
+	@BeforeClass
+	public static void init() {
+		testService = new DaytimeTypeService();
+	}
+
+	@Test
+	public void testGetDaytimeType() {
```

Comment:
same as next one - it should be one test per tested case. Let's discuss it tomorrow.